recipes/ncl/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "6.3.0" %}
+
+package:
+    name: ncl
+    version: {{ version }}
+
+source:
+    fn: ncl-{{ version }}.tar.gz
+    url: https://github.com/NCAR/ncl/archive/{{ version }}.tar.gz
+    sha256: d2cd8a1de1c498c1832f0765113ad58086d1d3b29bf5d2cc9b1ba60f1919951c
+
+build:
+    number: 0
+    skip: True  # [win]
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - gcc
+        - libnetcdf 4.4.*
+        - proj4 4.9.3
+        - gdal 2.1.*
+        - g2clib 1.6.*
+        - hdfeos2
+        - hdfeos5
+        - jasper
+        - libpng
+        - jpeg
```

Comment:
To be consistent with the rest of `conda-forge` stack this needs to be `jpeg 9*`. There are more pinning require and you can check them in https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies